test/source/tests/decrypt.ts

```diff
@@ -420,6 +420,15 @@ export const defineDecryptTests = (testVariant: TestVariant, testWithBrowser: Te
       expect(content).to.include('-----BEGIN PGP MESSAGE-----Version: FlowCrypt 7.4.2 Gmail\nEncryptionComment: Seamlessly send and receive encrypted\nemailwcFMA0taL/zmLZUBAQ/+Kj48OQND');
     }));
 
+    ava.default('decrypt - inbox - prevent tabnabbing in PGP/MIME links', testWithBrowser('compatibility', async (t, browser) => {
+      const inboxPage = await browser.newPage(t, 'chrome/settings/inbox/inbox.htm?acctEmail=flowcrypt.compatibility%40gmail.com&threadId=174124764bcc0494');
+      await inboxPage.waitAll('iframe.pgp_block');
+      const pgpBlock = await inboxPage.getFrame(['pgp_block.htm']);
+      await pgpBlock.waitForSelTestState('ready');
+      const htmlContent = await pgpBlock.readHtml('#pgp_block');
+      expect(htmlContent).to.include('rel="noopener noreferrer"');
```

Comment:
This tests the header but not the actual goal. If tabnabbing allows for a particular vulnerability - like clicking a link in the message to take over the original frame - then we should be clicking the link in the test, to see if that will happen or not.

It would be also good, afterwards, to see that once you have such test, if you remove the tabnabbing fix, the test should actually fail.

Reply:
I think it's worth putting both, because Chromium could add some kind of feature that makes window.opener null in this scenario, yet we'd still want to make sure the `rel="noopener noreferrer"` attribute is present to protect users on other browsers or older versions. 

I added a test that does as you suggest, and it works! If I disable the antitabnabbing mechanism, both of these new tests fail. When I readd it, they both succeed. 

*However*, my implementation of this test is somewhat of a mess, and it doesn't merit accession into the codebase as-is. Mostly due to my noob level Puppeteer skills. I was wondering if you perhaps have tips for how I might improve my new `window.opener` test?

Reply:
I think you've done it well. It does look a bit messy, but if it properly tests the intention, I'm ok to merge it.